djangocms_versioning_filer/fields/file.py

```diff
@@ -112,7 +114,20 @@ def widget_attrs(self, widget):
         widget.required = self.required
         return {}
 
+    def clean(self, value):
+        value = super().clean(value)
+
+        if value is not None and self.required is False:
+            published_exists = Version.objects.filter_by_grouper(value).filter(
+                state="published"
+            ).exists()
+            if not published_exists:
+                raise ValidationError("No published version currently exists, publish it first")
```

Comment:
This now works although you should be able to select a draft in edit mode. This coul dbe picked up after this PR.